### ISSUE TITLE
late-fusion

### ISSUE THREAD

**Issue by dsj1840923452** [Finspire13/CMCS-Temporal-Action-Localization#12]
Hi, excuse me, I recently ran through the code you provided. According to the I3D features in the paper, you finally adopted late-fusion, but I detected that the flow alone is even higher than that of late-fusion. What's the reason?thank you

**Comment by Finspire13** [comment]
Yes, we also found that results of only flow and late-fusion are similar for I3D (see supplementary).
Maybe you can try smaller fusion weight for RGB.

**Comment by dsj1840923452** [comment]
> Yes, we also found that results of only flow and late-fusion are similar for I3D (see supplementary).
> Maybe you can try smaller fusion weight for RGB.

Ok, thanks for your reply

**Comment by dsj1840923452** [comment]
> > Yes, we also found that results of only flow and late-fusion are similar for I3D (see supplementary).
> > Maybe you can try smaller fusion weight for RGB.
> 
> Ok, thanks for your reply

Hello, another question, is the mAp of the training set and the result of the test set similar, is that right?thank you
